lib/puppet/provider/mysql.rb

```diff
@@ -77,39 +75,33 @@ def self.cmd_table(table)
     table_string = ''
 
     # We can't escape *.* so special case this.
-    if table == '*.*'
-      table_string << '*.*'
-    # Special case also for PROCEDURES
-    elsif table.start_with?('PROCEDURE ')
-      table_string << table.sub(/^PROCEDURE (.*)(\..*)/, 'PROCEDURE `\1`\2')
-    else
-      table_string << table.sub(/^(.*)(\..*)/, '`\1`\2')
-    end
+    table_string << if table == '*.*'
+                      '*.*'
+                    # Special case also for PROCEDURES
+                    elsif table.start_with?('PROCEDURE ')
+                      table.sub(%r{^PROCEDURE (.*)(\..*)}, 'PROCEDURE `\1`\2')
+                    else
+                      table.sub(%r{^(.*)(\..*)}, '`\1`\2')
+                    end
     table_string
```

Comment:
do we need this if we're changing to the rubocop'd syntax?

Reply:
Not sure what you mean here? 
Updated with all your other comments 👍

Reply:
we can fix this later if needs be